commit 11b41853eb2047ad2f53ee51ec5872f5ba701fa7
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Mon Feb 8 13:24:25 2016 -0600

    Don't display warning if unable to connect when logged in via ssh
    
    https://bugzilla.gnome.org/show_bug.cgi?id=761600

diff --git a/atspi/atspi-misc.c b/atspi/atspi-misc.c
index 075a968..3a1d174 100644
--- a/atspi/atspi-misc.c
+++ b/atspi/atspi-misc.c
@@ -1588,7 +1588,8 @@ atspi_get_a11y_bus (void)
 
   if (!a11y_bus)
     {
-      g_warning ("Couldn't connect to accessibility bus: %s", error.message);
+      if (!g_getenv("SSH_CONNECTION"))
+        g_warning ("Couldn't connect to accessibility bus: %s", error.message);
       dbus_error_free (&error);
       return NULL;
     }
